osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -87,30 +87,36 @@ private void load(OsuColour colours, GameplayClock clock)
             };
         }
 
-        private const double skip_required_cutoff = 3000;
         private const double fade_time = 300;
 
-        private double beginFadeTime => startTime - skip_required_cutoff - fade_time;
+        /// <summary>
+        /// Represents minimum of secs required for gameplay start time in order to run skip overlay.
+        /// Since gameplay start time is 2 secs before beatmap's first object start time, in order to
+        /// use skip overlay if intro delay is >= 1 sec, we must substract 2 secs from 1, which is -1.
```

Comment:
what is intro delay 1 second? where is this coming from?

Reply:
I chose that value based on my experience, intuition, and tests that resulted to work right, having in mind idea of fixing mentioned issue. If u want to change it, I have nothing to comment, since it deppends on how u like the game to be, rather than having back-end impact.